src/components/Dialog.js

```diff
@@ -14,17 +37,5 @@ export default function Dialog(type, options) {
     }
   }
 
-  if (type == 'Open') {
-    return libui.UiDialogs.openFile(CURRENT_WINDOW);
```

Comment:
You're passing CURRENT_WINDOW, even though it is a global variable.

Reply:
This was already this way and is not part of this refactor.